src/components/navpanel.tsx

```diff
@@ -33,18 +33,20 @@ export const NavPanel = ({
 
   return (
     <TooltipProvider delayDuration={0}>
+      {/* Panel Group Left/Right */}
       <ResizablePanelGroup
         direction="horizontal"
         onLayout={(sizes: number[]) => {
           document.cookie = `react-resizable-panels:layout=${JSON.stringify(sizes)}`;
         }}
         className="h-full items-stretch"
       >
+        {/* Left Panel  */}
         <ResizablePanel
           defaultSize={defaultLayout[0]}
           collapsible={true}
           minSize={10}
-          maxSize={20}
+          maxSize={17}
```

Comment:
@iamtheozzy - Updated the maxSize so the user is not able to expand their sidebar after rendering. Do you think this is a better user experience?

Reply:
```suggestion
          maxSize={18}
```